skipchain/skipchain.go

```diff
@@ -1252,6 +1249,14 @@ func (s *Service) tryLoad() error {
 	return nil
 }
 
+// sliceToArr does what the name suggests, we need it to turn a slice into
```

Comment:
👍 for code non-reuse. I hate little util packages with this kind of crap in them. :)

(PS: This is not even sarcastic. Leave that right how it is!)